lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -236,16 +236,17 @@ defmodule ExUnit.Formatter do
 
   defp format_stacktrace(stacktrace, test_case, test, color) do
     extra_info("stacktrace:", color) <>
-      Enum.map_join(stacktrace,
-        fn(s) -> stacktrace_info format_stacktrace_entry(s, test_case, test), color end)
+      Enum.map_join(stacktrace, fn entry ->
+        stacktrace_info format_stacktrace_entry(entry, test_case, test), color
+      end)
   end
 
   defp format_stacktrace_entry({test_case, test, _, location}, test_case, test) do
-    "#{location[:file]}:#{location[:line]}"
+    format_file_line(location[:file], location[:line], " (file)")
```

Comment:
It was needed because of the `suffix`ing inside `Exception.format_file_line/3`, I have changed it to allow `nil` suffixes.